packages/smart-contracts/scripts/deploy-payments.ts

```diff
@@ -326,12 +320,11 @@ export async function deployAllPaymentContracts(
     );
     addToResult(swapConversionResult);
 
-    // EthereumFeeProxy
-    const { address: ethFeeProxyAddress } = await runEasyDeployment({
-      contractName: 'EthereumFeeProxy',
-      artifact: ethereumFeeProxyArtifact,
-      nonceCondition: NONCE_BATCH_3 + 1,
-    });
+    // Compute EthereumFeeProxy address (CREATE2)
+    const ethFeeProxyAddress = await computeCreate2DeploymentAddress(
+      { contract: 'EthereumFeeProxy' },
+      hre,
+    );
 
```

Comment:
Compute the address of EthereumFeeProxy so we can use it while deploying the EthConversionProxy contract